tests/unit/single_table/privacy/test_dcr_baseline_protection.py

```diff
@@ -169,6 +169,19 @@ def test_generate_random_data_different(self):
         with pytest.raises(AssertionError):
             pd.testing.assert_frame_equal(random_data_1, random_data_2)
 
+    def test__generate_random_data_with_seed(self):
+        """Test the generated data with a seed is the same every run."""
+        # Setup
+        real_data = pd.DataFrame({'float_col': [1.0, 1000.0, 500.0], 'cat_col': ['A', 'B', 'C']})
+        DCRBaselineProtection._seed = 5
```

Comment:
I think it's best to patch the seed here; otherwise, it might set the seed for all the other tests